src/js/course-list.js

```diff
@@ -7,8 +7,9 @@ $(() => {
     /*
         Click event to sort the course list
      */
-    $('#course-list th:not(:last)').on('click', function() {
-        var isAscending = (isDescending = false);
+    $('#course-list th:not(:last)').on('click', function () {
+        var isAscending = false,
+            isDescending = false;
```

Comment:
@vatz88 will need you to approve this. This is the only change I made, the rest was done by the prettier.